### PR TITLE
First final version

### PR DESCRIPTION
**Pull request by devchristiansimon**
Please check if you found errors or wrong a11y's.
im aware that the site title is still "document" and will fix it with the next commit

### CODE REVIEW

**Comment by devchristiansimon** [comment]
i upload it on my webspace: https://personal.christest.de/
pls watch it after reading the code

File: index.html [hunk 1]
```diff
@@ -5,9 +5,80 @@
...
+        <article class ="designArtikel">
+          <h3>Beim Bootcamp angemeldet</h3>
+          <p>In einem intensiven Auswahlverfahren konnte ich einen begehrten Platz bei den "Neuen Fischen" ergattern</p>
+        </article>
+
+      </section>
+      <section id="kontakt">
+        <h2>Sie wollen mich kontaktieren?</h2>
+        <article class ="designArtikel">
+          <h3>Per Telefon</h3>
```
Comment (PReichetanz) [inline index.html:41]: Please consider  the address HTML element for this content.

File: index.html [hunk 2]
```diff
@@ -5,9 +5,80 @@
...
+          <p>+49 xxxxxxxxxxxx</p>
+        </article>
+        <article class ="designArtikel">
```
Comment (PReichetanz) [inline index.html:44]: Please English naming for class names as well.

File: index.html [hunk 3]
```diff
@@ -5,9 +5,80 @@
...
+            <li>Teamarbeit</li>
+            <li>Lebensweisheiten</li>
+            <li>Kann gut zuhören</li>
+          </ul>
+        </article>
+      </section>
+    </main>
+    <footer>
+      <p>Hier folgt in Kürze das Impressum</p>
+      <p>Copyright by Simon Design 2022</p>
```
Comment (PReichetanz) [inline index.html:80]: Please consider the small HTML element for the copyright information.

File: index.html [hunk 4]
```diff
@@ -5,9 +5,80 @@
...
+      <p> <a href="#header">Hier geht es zurück nach oben</a></p>
```
Comment (PReichetanz) [inline index.html:81]:
```suggestion
      <a href="#header">Hier geht es zurück nach oben</a>
```

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+    flex-direction: column;
+    margin: 0;
+    align-items: center;
+    font-family: 'Roboto', sans-serif;
+}
+
+h2 {
+    width: 100%;
+    text-align: center;
+}
```
Comment (PReichetanz) [inline styles.css:16]:
```suggestion
}

```

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+nav {
+    height: 50px;
+    background-color:rgb(230, 249, 253);
+    width: 100%;
+    text-align: center;
+    display: flex;
+
+}
```
Comment (PReichetanz) [inline styles.css:24]:
```suggestion
}

```

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+header {
+    background-color: ghostwhite;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    
+}
```
Comment (PReichetanz) [inline styles.css:39]:
```suggestion
}

```
Looks like the prettier extension for your visual studio code is not working on saving the files.

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+
+.defaultDesign {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: center;
+    padding: 30px;
+    gap: 20px;
+}
+.designArtikel {
```
Comment (PReichetanz) [inline styles.css:65]:
```suggestion
.article {
```

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+    width: 400px;
+    padding: 20px;
+    background-color: ghostwhite;
+    box-shadow: 0 0 30px grey;
+}
+
+
```
Comment (PReichetanz) [inline styles.css:72]:
```suggestion
```

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+#kontakt {
```
Comment (PReichetanz) [inline styles.css:73]: Hint: Next week, we are going to explain why styling via ids is not a good idea; better use classes instead.

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+    flex-direction: row;
+    justify-content: center;
+    padding: 30px;
+    gap: 20px;
+}
+
+#kontakt > .designArtikel {
+    background-color: rgb(230, 249, 253);
+}
+
```
Comment (PReichetanz) [inline styles.css:86]: Please remove unused lines

**Review by PReichetanz — COMMENTED** [review]
Really nice work! I left some ideas how to further improve your code 🚀

File: index.html [hunk 12]
```diff
@@ -5,9 +5,80 @@
...
+    </nav>
+    <header id="header">
+      <h1>Das ist Christian Simon</h1>
+      <img src="./images/ich.png" alt="handsome man and owner of this website">
+      <q><em>Erfolg wird durch epische Zitate zur Realität!</em></q>
+    </header>
+    <main>
+      <section id="erfolge" class="defaultDesign">
+        <h2>Bisherige Erfolge</h2>
+        <article class ="designArtikel"><h3>Geburt</h3>
```
[the remark this answers is not in the document]
Comment (devchristiansimon) [reply index.html:25]: @PReichetanz  is the id "erfolge" here good or bad? in the one hand its the name of the german section, in the other hand its a german word and not english?

File: index.html [hunk 13]
```diff
@@ -5,9 +5,80 @@
...
-    <h1>YOYOYOYOYOOOOYOOOOOO i am chris simon</h1>
+    <nav>
+      <a href="#erfolge">Erfolge</a>
+      <a href="#kontakt">Kontakt</a>
+      <a href="#skills">Skills</a>
+    </nav>
+    <header id="header">
+      <h1>Das ist Christian Simon</h1>
+      <img src="./images/ich.png" alt="handsome man and owner of this website">
+      <q><em>Erfolg wird durch epische Zitate zur Realität!</em></q>
```
[the remark this answers is not in the document]
Comment (devchristiansimon) [reply index.html:20]: q tag doesn't make the italic style, i tried

File: index.html [hunk 14]
```diff
@@ -5,9 +5,80 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="/styles.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet"> 
     <title>Document</title>
   </head>
   <body>
-    <h1>YOYOYOYOYOOOOYOOOOOO i am chris simon</h1>
```
Comment (chris-yooo) [inline index.html:11]: this line <3

File: index.html [hunk 15]
```diff
@@ -5,9 +5,80 @@
...
+    <nav>
+      <a href="#erfolge">Erfolge</a>
+      <a href="#kontakt">Kontakt</a>
+      <a href="#skills">Skills</a>
+    </nav>
+    <header id="header">
+      <h1>Das ist Christian Simon</h1>
+      <img src="./images/christian_simon_profil.png" alt="handsome man and owner of this website">
+      <q><em>Erfolg wird durch epische Zitate zur Realität!</em></q>
```
Comment (chris-yooo) [inline index.html:20]: ^^ cool

File: index.html [hunk 16]
```diff
@@ -5,9 +5,80 @@
...
+    </header>
+    <main>
+      <section id="erfolge" class="defaultDesign">
+        <h2>Bisherige Erfolge</h2>
+        <article class ="article">
+          <h3>Geburt</h3>
+        <p>Im Jahr 1986 erfolgreich geboren.</p>
```
Comment (chris-yooo) [inline index.html:27]: xD

File: index.html [hunk 17]
```diff
@@ -5,9 +5,80 @@
...
+        </article>
+        <article class ="article">
+          <h3>Schule beendet</h3>
+          <p>Nach jahrelangem intensiven Trainings konnte ich erfolgreich die Schule beenden!</p>
+        </article>
+        <article class ="article">
+          <h3>Beim Bootcamp angemeldet</h3>
+          <p>In einem intensiven Auswahlverfahren konnte ich einen begehrten Platz bei den "Neuen Fischen" ergattern</p>
```
Comment (chris-yooo) [inline index.html:35]: in this line the dot . @ end is missing.

File: index.html (hunk 4 above)
Comment (chris-yooo) [reply index.html:81]: 1+

File: styles.css [hunk 18]
```diff
@@ -0,0 +1,104 @@
...
+    width: 400px;
+    padding: 20px;
+    background-color: ghostwhite;
+    box-shadow: 0 0 30px grey;
+}
+
+#kontakt {
+    background-color: ghostwhite;
+    display: flex;
+    flex-wrap: wrap;
```
Comment (chris-yooo) [inline styles.css:74]: here you can use flex-flow: row wrap

File: styles.css [hunk 19]
```diff
@@ -0,0 +1,104 @@
...
+
+#kontakt > .article {
+    background-color: rgb(230, 249, 253);
+}
+
+footer {
+    width: 100%;
+    background-color: rgb(127, 128, 129);
+    display: flex;
+    flex-flow: row wrap;
```
Comment (chris-yooo) [inline styles.css:89]: here you used it :D

File: index.html [hunk 20]
```diff
@@ -5,9 +5,80 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="/styles.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
```
Comment (chris-yooo) [inline index.html:8]: <3

**Review by chris-yooo — COMMENTED** [review]
finish :)

File: index.html (hunk 13 above)
[the remark this answers is not in the document]
Comment (SLS-BLN) [reply index.html:20]: you can change the font-style later with CSS

File: index.html (hunk 20 above)
Comment (SLS-BLN) [reply index.html:8]: I would recommend adding 500 as a third font weight - I think that's what Chris said (<3)

Comment (devchristiansimon) [reply index.html:8]: ryan-reynolds-but-why

File: index.html (hunk 12 above)
[the remark this answers is not in the document]
Comment (PReichetanz) [reply index.html]: Even if the content of the page is German, we use English for the entire code; so "erfolge" is not a good name as you have already pointed out 😉

File: styles.css [hunk 21]
```diff
@@ -0,0 +1,104 @@
...
+    gap: 20px;
+}
+.article {
+    width: 400px;
+    padding: 20px;
+    background-color: ghostwhite;
+    box-shadow: 0 0 30px grey;
+}
+
+#kontakt {
```
Comment (actyralf) [inline styles.css:71]: you can use `.defaultDesign` to style the `address` tag. please use `.nth-child(even)` to achieve the alternating background colors.

File: index.html (hunk 17 above)
Comment (Lee1887) [inline index.html:35]: "Einen der begehrten Plätze """"""ergattern. " Perhaps a minor tweak to make the text more readable.

**Pull request merged**